crates/bevy_ui/src/geometry.rs

```diff
@@ -155,6 +156,22 @@ impl DivAssign<f32> for Val {
     }
 }
 
+impl Display for Val {
+    fn fmt(&self, f: &mut std::fmt::Formatter<'_>) -> std::fmt::Result {
+        let (value, suffix) = match self {
+            Val::Auto => return write!(f, "auto"),
+            Val::Px(value) => (value, "px"),
+            Val::Percent(value) => (value, "%"),
+            Val::Vw(value) => (value, "vw"),
+            Val::Vh(value) => (value, "vh"),
+            Val::VMin(value) => (value, "vmin"),
+            Val::VMax(value) => (value, "vmax"),
+        };
+        value.fmt(f)?;
+        write!(f, "{suffix}")
```

Comment:
Would this be equivalent or does it result in a different output?
```suggestion
        write!(f, "{value}{suffix}")
```

Reply:
Yes, the first version allows you to specify formatting options. So with `value.fmt(f)?`:
```rust
println!("{:.2}", Val::Px(0.5689));
```
has output
```
0.57
```
whereas with `write!(f, "{value}{suffix}")` it will output:
```
0.5689
```

Reply:
That's good to know, thanks!